cloud/docker/docker.py

```diff
@@ -772,6 +778,7 @@ def get_start_params(self):
             'privileged': self.module.params.get('privileged'),
             'links': self.links,
             'network_mode': self.module.params.get('net'),
+            'read_only': self.module.params.get('read_only'),
```

Comment:
i would still have the version check, people seem to be running many diff versions of docker.py ... makes this module 'fun'